<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmTcpServer</class>
 <widget class="QWidget" name="frmTcpServer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>519</width>
    <height>481</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>900</y>
     <width>606</width>
     <height>23</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="layTcpServer">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
   </layout>
  </widget>
  <widget class="QPushButton" name="btnUpgrate">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>450</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>80</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>80</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="text">
    <string>在线升级</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="txtMain">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>11</y>
     <width>321</width>
     <height>341</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>11</y>
     <width>170</width>
     <height>341</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>170</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>170</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <property name="leftMargin">
     <number>9</number>
    </property>
    <property name="topMargin">
     <number>9</number>
    </property>
    <property name="rightMargin">
     <number>9</number>
    </property>
    <property name="bottomMargin">
     <number>9</number>
    </property>
    <item>
     <widget class="QLabel" name="labListenPort">
      <property name="text">
       <string>监听端口</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="txtListenPort"/>
    </item>
    <item>
     <widget class="QPushButton" name="btnListen">
      <property name="text">
       <string>监听</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnSave">
      <property name="text">
       <string>保存</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnClear">
      <property name="text">
       <string>清空</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="labCount">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>25</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="text">
       <string>共 0 个连接</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QListWidget" name="listWidget"/>
    </item>
    <item>
     <widget class="QCheckBox" name="ckAll">
      <property name="text">
       <string>对全部客户端发送</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="btnBrowser">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>360</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>80</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>80</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="text">
    <string>浏览</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnConnect">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>450</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>80</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>80</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="text">
    <string>连接设备</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="browseFileEdit">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>360</y>
     <width>351</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="labListenPort_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>360</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>升级文件</string>
   </property>
  </widget>
  <widget class="QLabel" name="labListenPort_4">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>390</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Bytes</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="fileSizeEdit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>390</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labListenPort_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>390</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>文件大小</string>
   </property>
  </widget>
  <widget class="QLabel" name="labListenPort_5">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>390</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>单次发送长度</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="oneSizeEdit">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>390</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labListenPort_6">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>390</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Bytes</string>
   </property>
  </widget>
  <widget class="QLabel" name="labListenPort_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>420</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>正发送第</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="sendingNumEdit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>420</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labListenPort_8">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>420</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>条</string>
   </property>
  </widget>
  <widget class="QLabel" name="labListenPort_9">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>420</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>共发送</string>
   </property>
  </widget>
  <widget class="QLabel" name="labListenPort_10">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>420</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>条</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="sendAllNumEdit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>420</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labListenPort_11">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>450</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>设备编号</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="termialAddrEdit">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>450</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="connectTerStatusLabe">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>450</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
